src/app/inventory-page/StoreBucket.tsx

```diff
@@ -108,15 +122,31 @@ const StoreBucketInner = memo(function StoreBucketInner({
         </StoreBucketDropTarget>
       )}
       <StoreBucketDropTarget
+        grouped={isVault && Boolean(vaultWeaponGroupingSetting)}
```

Comment:
If you're only using this as a boolean you should probably select a boolean, e.g. `vaultWeaponGroupingEnabledSelector`

Reply:
The actual value is used elsewhere, I just need to know if it's truthy in this case.

Reply:
Right - it's a nitpick, but `useSelector` will trigger a re-render whenever the output value of the selector changes - you have only two useful values (true/false) but your output can be one of many values. It doesn't really matter in practice, it's just good habit.

Reply:
Oh, I see what you mean. I can add another selector. 😊